content/participants/ttjc@ritbikbharti/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Ritbik Bharti
+type: "participant"
+name: "Ritbik Bharti"
+portfolio: "https://tanaypratap.com"
```

Comment:
Hey! @ritbikbharti  Please update your portfolio link here and if you dont have one leave it blank.

Reply:
Thanks! I have updated.